OpenSim/Tests/testIterators/testIterators.cpp

```diff
@@ -150,7 +150,11 @@ int main()
         // to recompose existing components, this will need continual updating. For example,
         // Joint's often add PhysicalOffsetFrames to handle what used to be baked in location
         // and orientation offsets.
-        ASSERT(numComponents == 77); 
+
+        // Updated below from 77 to 65 when pennation and activation models in
+        // Thelen2003Muscle were converted into private (non-serialized)
+        // subcomponents (2 subcomponents x 6 muscles).
+        ASSERT(numComponents == 65); 
```

Comment:
I think this is a design question that falls beyond the scope of this PR. Perhaps a new issue should be created for this discussion?